test/Error/rlexe.xml

```diff
@@ -30,6 +30,8 @@
     <default>
       <files>stack.js</files>
       <baseline>stack.baseline</baseline>
+      <!-- xplat-todo: Understand why this test is failing on OS X -->
+      <tags>exclude_mac</tags>
```

Comment:
Cool, thanks @obastemur- I'll rebase this change and remove the exclude_mac bit once you've merged your change in.